tracer/src/Datadog.Trace/Debugger/LiveDebuggerFactory.cs

```diff
@@ -28,19 +28,16 @@ internal class LiveDebuggerFactory
 {
     private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor(typeof(LiveDebuggerFactory));
 
-    public static LiveDebugger Create(IDiscoveryService discoveryService, IRcmSubscriptionManager remoteConfigurationManager, ImmutableTracerSettings tracerSettings, string serviceName, ITelemetryController telemetry)
+    public static LiveDebugger Create(IDiscoveryService discoveryService, IRcmSubscriptionManager remoteConfigurationManager, ImmutableTracerSettings tracerSettings, string serviceName, ITelemetryController telemetry, DebuggerSettings debuggerSettings)
     {
-        var settings = DebuggerSettings.FromDefaultSource();
-        if (!settings.Enabled)
+        if (!debuggerSettings.Enabled)
         {
-            telemetry.ProductChanged(TelemetryProductType.DynamicInstrumentation, enabled: false, error: null);
-            Log.Information("Live Debugger is disabled. To enable it, please set DD_DYNAMIC_INSTRUMENTATION_ENABLED environment variable to 'true'.");
-            return LiveDebugger.Create(settings, string.Empty, null, null, null, null, null, null, null);
+            return LiveDebugger.Create(debuggerSettings, string.Empty, null, null, null, null, null, null, null);
```

Comment:
I think we should keep the log line here that the live debugger is disabled because if we got to this point then that means that Remote Configuration is available but Live Debugger is disabled

Reply:
Yeah, you're right, it was a mistake trying to move it to `Instrumentation`!